packages/augur-ui/src/modules/market-cards/common.tsx

```diff
@@ -82,30 +88,121 @@ export interface OutcomeGroupProps {
 	scalarDenomination?: string;
 	min?: number;
 	max?: number;
-  	percent?: number;
+  lastPrice?: number;
 }
 
-export const OutcomeGroup = (props: OutcomeGroupProps) => (
-  	<div className={classNames(Styles.OutcomeGroup, {
+export const OutcomeGroup = (props: OutcomeGroupProps) => {
+  let outcomesShow = props.outcomes.filter(outcome => outcome.isTradable);
+  const removedInvalid = outcomesShow.splice(0, 1)[0];
+  outcomesShow.splice(2, 0, removedInvalid);
+
+  return (
+    <div className={classNames(Styles.OutcomeGroup, {
 			[Styles.Categorical]: props.marketType === CATEGORICAL, 
 			[Styles.Scalar]: props.marketType === SCALAR, 
 			[Styles.YesNo]: props.marketType === YES_NO
 		})}>
   		{props.marketType === SCALAR &&
-  			<ScalarOutcome
-  				min={props.min}
-  				max={props.max}
-  				percent={props.percent}
-  				scalarDenomination={props.scalarDenomination}
-  			/>
+        <>
+    			<ScalarOutcome
+    				min={props.min}
+    				max={props.max}
+    				lastPricePercent={props.lastPricePercent}
+    				scalarDenomination={props.scalarDenomination}
+    			/>
+          <Outcome
+            description={removedInvalid.description}
+            lastPricePercent={removedInvalid.lastPricePercent}
+            invalid={true}
+            index={0}
+          /> 
+        </>
   		}
-	  	{props.outcomes.map((outcome: Outcome, index: number) =>
-	  		(!props.expanded && index < 2 || props.expanded) && <Outcome
-	  			name={outcome.name}
-	  			percent={outcome.percent}
-	  			invalid={outcome.invalid}
+	  	{props.marketType !== SCALAR && outcomesShow.map((outcome: Outcome, index: number) =>
+	  		(!props.expanded && index < 3 || (props.expanded || props.marketType === YES_NO)) && <Outcome
+	  			description={outcome.description}
+	  			lastPricePercent={outcome.lastPricePercent}
+	  			invalid={outcome.description === "Invalid"}
```

Comment:
outcome 0 is always invalid, user can create their own outcome named `Invalid` that would screw this up possible.